website/.vitepress/meta.ts

```diff
@@ -0,0 +1,5 @@
+export const taskName = 'Task';
+export const taskDescription =
+  'A fast, cross-platform build tool inspired by Make, designed for modern workflows.';
```

Comment:
Is this officially our new tagline? If so, we should use this everywhere (GitHub/socials etc). Otherwise, this should stay the same as it is today.

Reply:
I haven't told you, I tried something more engaging
But I am perfectly fine to stay with the current's one (that's why it's stored in the variable, easy to change)

What are your thoughts on it @andreynering and @pd93  ?